contracts/assemblyscript/flipper/assembly/index.ts

```diff
@@ -21,46 +21,44 @@ enum Action {
 }
 
 function handle(input: Uint8Array): Uint8Array { // vec<u8>
-  const value: Uint8Array = new Uint8Array(0);
-  const flipper: Uint8Array = getStorage(FLIPPER_KEY);
-  const dataFlipper: DataView = new DataView(flipper.buffer);
+  const value = new Uint8Array(0);
+  const flipper = getStorage(FLIPPER_KEY);
+  const dataFlipper = new DataView(flipper.buffer);
   const flipperValue: u8 = dataFlipper.byteLength ? dataFlipper.getUint8(0) : 0;
 
   // Get action from first byte of the input U8A
   switch (input[0]) {
-    case Action.Flip:
-      const newFlipperBool: bool = !<bool>flipperValue;
-      const newFlipperU8a: Uint8Array = new Uint8Array(1);
-      newFlipperU8a[0] = newFlipperBool;
-      setStorage(FLIPPER_KEY, newFlipperU8a)
+    case Action.Flip: {
```

Comment:
Arrows in a switch case are rather non-standard and not necessary so I'd prefer to not add them here & in the other example.

Reply:
I removed brackets